content/integrations/sumologic.md

```diff
@@ -13,11 +13,12 @@ Enable the Sumo Logic integration to:
 
 ## Installation
 
-There are two parts to the SumoLogic integration, based on the direction of information between the products. You can send alerts and events from Datadog to Sumologic and you can have SumoLogic send data to Datadog.
+Datadog integration with SumoLogic is two-part: send alerts and events from Datadog to Sumologic, or have SumoLogic send data to Datadog. You can enable one or both.
```

Comment:
i would use that to describe the rds integration (with three instead of two) since there are 3 different ways that you may choose and you probably arent going to use all of them

this has two parts or components and you are probably going to use both. Maybe just eliminate it: The datadog integration with sumologic can send...blah blah